packages/create-plugin/templates/github/ci/.github/workflows/ci.yml

```diff
@@ -94,3 +96,33 @@ jobs:
           name: cypress-videos
           path: cypress/videos
           retention-days: 5
+
+      - name: Sign plugin
+        run: {{ packageManagerName }} run sign
+        if: \${{ env.GRAFANA_ACCESS_POLICY_TOKEN != '' }}
+
+      - name: Get plugin metadata
+        id: metadata
+        run: |
+          sudo apt-get install jq
+
+          export GRAFANA_PLUGIN_ID=$(cat dist/plugin.json | jq -r .id)
+          export GRAFANA_PLUGIN_VERSION=$(cat dist/plugin.json | jq -r .info.version)
+          export GRAFANA_PLUGIN_ARTIFACT=${GRAFANA_PLUGIN_ID}-${GRAFANA_PLUGIN_VERSION}.zip
+
+          echo "plugin-id=${GRAFANA_PLUGIN_ID}" >> $GITHUB_OUTPUT
+          echo "plugin-version=${GRAFANA_PLUGIN_VERSION}" >> $GITHUB_OUTPUT
+          echo "archive=${GRAFANA_PLUGIN_ARTIFACT}" >> $GITHUB_OUTPUT
+
+      - name: Package plugin
+        id: package-plugin
+        run: |
+          mv dist \${{ steps.metadata.outputs.plugin-id }}
+          zip \${{ steps.metadata.outputs.archive }} \${{ steps.metadata.outputs.plugin-id }} -r
+
+      - name: Archive Build
+        uses: actions/upload-artifact@v3
+        with:
+          name: \${{ steps.metadata.outputs.plugin-id }}-\${{ steps.metadata.outputs.plugin-version }}
+          path: \${{ steps.metadata.outputs.plugin-id }}
```

Comment:
Nit: is the escaping of the dollar signs necessary here?

Reply:
double checked this and fixed the escapes - they needed to be after the $ not before, the generated file now matches what is expected